lib/tasks/eventboss.rake

```diff
@@ -6,6 +6,9 @@ namespace :eventboss do
     task :reload, [:event_name, :source_app, :max_messages] do |task, args|
       source_app = args[:source_app]
       event_name = args[:event_name]
+      start_time = Time.now
```

Comment:
Not sure if we need it - the time will be logged in kibana anyway

Reply:
We need this if we want to calculate how long is rake task running. But if we don't need this log - I can remove it. 

But I can remove logging start time and leave calculating total time for task execution.

Reply:
Start time log removed, but there is still question do we need log for time execution of rake task.